modules/mcaptcha.nix

```
@@ -0,0 +1,279 @@
+{
+  config,
+  lib,
+  options,
+  pkgs,
+  ...
+}:
+with builtins;
+with lib; let
+  cfg = config.services.mcaptcha;
```

I'll use that in comments below.

```suggestion
  cfg = config.services.mcaptcha;
  opt = options.services.mcaptcha;
```

---

Took your advice.